bibliopixel/remote/trigger_process.py

```diff
@@ -1,9 +1,9 @@
 from .. util import importer
 
 
-def run_trigger(typename, q, configs):
+def run_trigger(typename, q, events, kwargs):
```

Comment:
Well, here's an example - why isn't `events` passed in as part of `kwargs`?  `events` isn't actually used in this method, so why isn't it passed through in `kwargs` without ever even being mentioned here?

Just a note though for future reference - don't make this change now, we don't know the final form - but this is the sort of thing I'm thinking.